internal/pkg/wirecli/wirek8s/wirek8s.go

```diff
@@ -14,32 +14,49 @@ limitations under the License.
 package wirek8s
 
 import (
-	"bytes"
 	"fmt"
 	"io/ioutil"
-	"os"
 	"path/filepath"
+	"sigs.k8s.io/kustomize/pkg/resource"
 	"strings"
 
 	"github.com/google/wire"
 	"github.com/spf13/cobra"
+
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/cli-runtime/pkg/kustomize"
+	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"sigs.k8s.io/cli-experimental/internal/pkg/clik8s"
+	"sigs.k8s.io/cli-experimental/internal/pkg/resourceconfig"
 	"sigs.k8s.io/cli-experimental/internal/pkg/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Could we avoid depending on controller-runtime here.
Iam envisioning this to be vendored in controller-runtime and not the other way around.